earthpy/tests/test_plot.py

```diff
@@ -0,0 +1,86 @@
+"""Tests for the plot module"""
+
+import numpy as np
+import pytest
+import earthpy.spatial as es
+import matplotlib as mpl
+
+
+
+""" General functions for matplotlib elements """
+
+
+# Create tuple
+tuups = (1, 2)
+
+im = np.random.randint(10, size=(2, 4, 5))
+single_band = im[0]
```

Comment:
The `single_band` object seems to only be used by one test (`test_num_titles`) - would it make sense to define it within that function, rather than in the global environment?

Reply:
fixed!!